gif.h

```diff
@@ -732,6 +732,61 @@ typedef struct
     bool firstFrame;
```

Comment:
Why do we include this in the existing gif.h header? Can we create a separate header to store fuzzer specific function calls?